src/main/java/net/mcreator/ui/modgui/BlockGUI.java

```diff
@@ -385,13 +386,12 @@ public BlockGUI(MCreator mcreator, ModElement modElement, boolean editingMode) {
 				hasGravity.setSelected(false);
 				rotationMode.setSelectedIndex(0);
 
-				if (blockBase.getSelectedItem().equals("Wall") || blockBase.getSelectedItem().equals("Fence")
-						|| blockBase.getSelectedItem().equals("TrapDoor") || blockBase.getSelectedItem().equals("Door")
-						|| blockBase.getSelectedItem().equals("FenceGate") || blockBase.getSelectedItem()
-						.equals("EndRod") || blockBase.getSelectedItem().equals("PressurePlate")) {
-					if (!isEditingMode()) {
+				if (!isEditingMode()) {

```

Comment:
As light opacity is ignored for block bases other than leaves, I think UI should disable this field for all bases but leaves to make the UI more clear on the fact opacity can't be controlled by the user in this case.

Reply:
It's not ignored by other block bases, only if the opacity is set to 0